.env.sample

```diff
@@ -39,6 +39,7 @@ SESSION_SECRET=
 # UTILISATION REPORT
 PORTAL_REPORT_DUE_DATE_BUSINESS_DAYS_FROM_START_OF_MONTH=10
 PORTAL_REPORTING_PERIOD_START_EMAIL_SCHEDULE='0 7 1 * *'
+PORTAL_REPORT_DUE_EMAIL_SCHEDULE='0 7 * * *'
```

Comment:
It doesn't affect local/this PR but do we have a time we want to send this given to us by product?

Reply:
I'll check 👍